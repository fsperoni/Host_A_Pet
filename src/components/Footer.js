import React from "react";
import '../styles/Footer.scss';

const Footer = () => {
  return (
    <footer className="footer mt-auto py-3">
      <p className="text-center my-auto">Â© Fabio Speroni 2022</p>
    </footer>
  )
}

export default Footer;