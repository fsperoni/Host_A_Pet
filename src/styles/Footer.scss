footer {
  height: 3rem;
  color: var(--white);
  background-color: var(--black);
}