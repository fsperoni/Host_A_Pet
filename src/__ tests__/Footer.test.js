import { render } from "@testing-library/react";
import Footer from "../components/Footer";

// smoke test
it('renders without crashing', function() {
  render(<Footer />);

});

//snapshot test
it("matches snapshot", function() {
  const {asFragment} = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});

it("should have author's name in the component.", function() {
  const { getByText } = render(<Footer />);
  expect(getByText("Â© Fabio Speroni 2022")).toBeInTheDocument();
});