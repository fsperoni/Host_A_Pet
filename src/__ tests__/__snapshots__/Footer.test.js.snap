// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer mt-auto py-3"
  >
    <p
      class="text-center my-auto"
    >
      Â© Fabio Speroni 2022
    </p>
  </footer>
</DocumentFragment>
`;
